src/v/kafka/server/handlers/topics/types.h

```diff
@@ -16,12 +16,31 @@
 #include "kafka/server/errors.h"
 #include "model/fundamental.h"
 #include "model/namespace.h"
+#include "utils/sstring_hash.h"
 
 #include <absl/container/flat_hash_map.h>
 #include <boost/algorithm/string.hpp>
 #include <boost/lexical_cast.hpp>
 
 namespace kafka {
+using config_map_t
```

Comment:
node hash?

Reply:
those map are really small, there is no benefit from using `node_hash_map` in here.

Reply:
a comment would be useful then.